database/migrations/2024_06_18_093200_ragnarok_consat_stop_mapping.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+/**
+ * Columns for stop place ID and name moved.
+ */
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('consat_calls', function (Blueprint $table) {
+            $table->char('nsr_stop_id', 64)->after('sequence')->comment('NSR quay ID');
+            $table->string('stop_name')->after('nsr_stop_id')->comment('NSR quay name');
```

Comment:
The comment should reflect that it is the stop name, not quay name (which usually is 01, 02, etc…)